### PR TITLE
added duedil to the community

### PR DESCRIPTION
**Pull request by jpantsjoha**
We're Using fluxCD at Duedil !
How? Read more on my latest blog post on this topic
ref https://medium.com/@jaroslav.pantsjoha/a39eb650576f

### CODE REVIEW

**Comment by jpantsjoha** [comment]
@trevrosen @squaremo this is ready for merge > Merely putting one more name to the list there. thanks

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ container images and config changes are propagated to the cluster.
 |[Canva](https://www.canva.com/)|[Chattigo](https://chattigo.com)|[Cloudlets](https://cloudlets.io)|
 |[Cloudogu GmbH](https://cloudogu.com)|[CLVR Cloud Ltd](https://clvr.cloud)|[College of William & Mary](https://www.wm.edu)|
 |[Control Plane](https://control-plane.io)|[Consorcio Chile](https://www.consorcio.cl)|[CrowdRiff](https://crowdriff.com)|
+|[Duedil](https://www.duedil.com/)
```
Comment (trevrosen) [inline README.md:57]: Do you need a `|` at the end of this line?

Comment (hiddeco) [reply README.md:57]: It needs to be merged in after `|[Cybrary](https://cybrary.it)|`, which means all other entries will move one position (either to the right, or one line lower).

**Comment by dholbach** [comment]
I took the liberty of applying your change in #3234 (and resort and reflow the table).

**Comment by dholbach** [comment]
And thanks for the original PR, @jpantsjoha

File: README.md (hunk 1 above)
Comment (jpantsjoha) [reply README.md:57]: We need a better formatting options for this. really is not clear to follow and format this.
i can see this is superseded by #3234

**Comment by jpantsjoha** [comment]
Thanks!

**Pull request closed without merging**